src/Nri/Ui/Css/V1.elm

```diff
@@ -0,0 +1,32 @@
+module Nri.Ui.Css.V1 exposing (styleIf, styleJust)
+
+{-|
+
+@docs styleIf, styleJust
+
+-}
+
+import Css exposing (Style, property)
+
+
+{-| Apply a style only if a condition is `True`.
+-}
+styleIf : (() -> Style) -> Bool -> Style
+styleIf view condition =
+    if condition then
+        view ()
+
+    else
+        property "-nri" "noop"
```

Comment:
this can be unhacked like so:

```suggestion
        Css.batch []
```

Reply:
Oh man I searched for this forever.  Amazing

Reply:
Or rather an approach that wasn't hacky.